test/library.cpp

```diff
@@ -376,7 +400,7 @@ TEST_F(LibraryTest, bookmarksSerializationTest)
 
     lib->addBookmark(createBookmark(bookId1, "a/url", "Article title1"));
     lib->addBookmark(createBookmark("invalid-bookmark-id", "another/url", "Unknown title"));
-    lib->addBookmark(createBookmark(bookId2, "a/url/2", "Article title2"));
+    lib->addBookmark(createBookmark(book2, "a/url/2", "Article title2"));
```

Comment:
If it's intentional that the bookmarks for book1 and book2 are created via book-id and the book object, respectively, it's better to have a comment making such a setup more obvious and explaining why it is important.

Reply:
It is intentional, I will add a comment.